common-cli/src/main/java/co/cask/common/cli/completers/StringsCompleter.java

```diff
@@ -36,13 +39,14 @@
   @Override
   protected Collection<String> getCandidates(String buffer) {
     ImmutableList.Builder<String> candidates = ImmutableList.builder();
-
+    ArrayList<String> mutableCandidates = new ArrayList<String>();
```

Comment:
for instantiation, use ``Lists.newArrayList()`` as opposed to ``new ArrayList<String>()``